dali/pipeline/data/types.cc

```diff
@@ -63,5 +66,30 @@ void TypeInfo::Copy<GPUBackend, GPUBackend>(void *dst,
   MemCopy(dst, src, n*size(), stream);
 }
 
+DALI_ADD_TO_TYPE_TABLE(uint8_t);
```

Comment:
Is it possible to connect it somehow with https://github.com/NVIDIA/DALI/blob/master/dali/pipeline/data/types.cc#L318 so the list of the registered types are not duplicated?

Reply:
Just define DALI_TYPENAME_REGISTERER to do the job.